contracts/contracts/core/BidderRegistry.sol

```diff
@@ -114,8 +116,11 @@ contract BidderRegistry is
             maxBidPerBlock[msg.sender][window] =
                 newLockedFunds /
                 WindowFromBlockNumber.BLOCKS_PER_WINDOW;
-
+            
             emit BidderRegistered(msg.sender, newLockedFunds, window);
+            emit BidderDepositedForWindow(msg.sender, window, amountToDeposit, newLockedFunds);
```

Comment:
Why do we need to emit `BidderRegistered` right before this new event, `BidderDepositedForWindow`? They seem to emit the same data

Reply:
The new event includes amountToDesposit (e.g the amount being deposited by the user) The old one only included the aggregate funds in the window. Both are being emited for backwards compatibility.

Reply:
This event is used by the DepositManager to keep track of balances of the bidder. So if we want to use this new event, there will be changes required on mev-commit side as well.